debugging/coreclr/dockerDebugConfigurationProvider.ts

```diff
@@ -100,7 +100,7 @@ export class DockerDebugConfigurationProvider implements DebugConfigurationProvi
 
         const { appFolder, resolvedAppFolder } = await this.inferAppFolder(folder, debugConfiguration);
 
-        const { appProject, resolvedAppProject } = await this.inferAppProject(folder, debugConfiguration, resolvedAppFolder);
+        const { resolvedAppProject } = await this.inferAppProject(folder, debugConfiguration, resolvedAppFolder);
```

Comment:
I don't see changes to the `inferAppProject` function to reflect this change.

Reply:
`inferAppProject` returns an object - removing `appProject` is essentially like removing a property name from an interface, which doesn't require any sort of change on the original object.